build.sbt

```diff
@@ -1,11 +1,16 @@
 import sbtassembly.AssemblyPlugin.defaultUniversalScript
 
-ThisBuild / scalaVersion := "2.13.6"
+ThisBuild / scalaVersion := "2.12.18"
```

Comment:
This is intentional, 2.13 brings 100 + deprecations, 
so first, get a clean 2.12 state, and then move on